src/main/java/org/mdbenefits/app/submission/conditions/HasMoreThan100OnHand.java

```diff
@@ -0,0 +1,18 @@
+package org.mdbenefits.app.submission.conditions;
+
+import formflow.library.config.submission.Condition;
+import formflow.library.data.Submission;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HasMoreThan100OnHand implements Condition {
+
+    @Override
+    public Boolean run(Submission submission) {
+        Map<String, Object> inputData = submission.getInputData();
+        return inputData.getOrDefault("householdMoneyOnHandLessThan100", "false")
```

Comment:
If they didn't answer/see this question, won't this still end up being truthy? Is that desired?

Reply:
This is a tough one, as I'm not sure which the default should be. Unset == truthy here, so it would look like they have more than 100 on hand. Thinking out loud here: 

If a default of "false" makes it default to truthy (HasMoreThan100OhHand = true), then the assumption is that they do have more than 100 on hand and they will get asked more about how much they have on hand.   

If a default to "true" makes it default to "falsey" (HasMoreThan100OnHand= false) then it looks like they have LESS than 100 on hand and we might skip the question of how much they have on hand - which may result in us not marking them as XSNAP.  

It seems like the first scenario is more in their favor -- assuming false and then the only extra is asking how much money on hand they have.  

Thoughts?